exercises/chapter8/test/Main.purs

```diff
@@ -67,20 +37,14 @@ main =
         test "4 elements"
           $ Assert.equal (Just 4)
           $ third [ 1, 2, 4, 3 ]
-      suite "Exercise 2 - function possibleSums" do
```

Comment:
Less maintenance if we don't use numbers. I just list the function name, since it's safe to assume each exercise asks you to write a function.